data/112/538/944/1/1125389441.geojson

```diff
@@ -3,6 +3,7 @@
   "type": "Feature",
   "properties": {
     "edtf:cessation":"uuuu",
+    "edtf:deprecated":"2024-01-19",
```

Comment:
This property should be fine, but I'll add in the `"edtf:superseded"` property, too.

Reply:
No, it should be `edtf:deprecated` (and the `wof:superseded_by` array populated by the new record) assuming that is the intention. As in: This record has been deprecated because it is a duplicate.

There have never been dedicated fields for when or why a record is deprecated. We could add those (like `edtf:superseded`) but the `edtf:deprecated` is the specific signal that a record is now considered bunk (and no longer updated).

Reply:
This is worth a quick wof-properties update so this doesn't get lost in this PR